ct/bookstack.sh

```diff
@@ -70,9 +70,10 @@ if [[ ! -f /opt/${APP}_version.txt ]] || [[ "${RELEASE}" != "$(cat /opt/${APP}_v
   unzip -q v${RELEASE}.zip
   mv BookStack-${RELEASE} /opt/bookstack
   mv /opt/.env /opt/bookstack/.env
-  COMPOSER_ALLOW_SUPERUSER=1 composer install --no-dev  &>/dev/null
-  php artisan key:generate &>/dev/null
-  php artisan migrate &>/dev/null
+  cd /opt/bookstack
+  composer install --no-dev --no-interaction &>/dev/null
```

Comment:
```suggestion
  COMPOSER_ALLOW_SUPERUSER=1 composer install --no-dev &>/dev/null
```

This was the original error
```
root@bookstack:~#  COMPOSER_ALLOW_SUPERUSER=1 composer install --no-dev
Composer could not find a composer.json file in /root
To initialize a project, please create a composer.json file. See https://getcomposer.org/basic-usage
```
adding `cd /opt/bookstack` fixes it and the original command runs fine after that.

We should keep `COMPOSER_ALLOW_SUPERUSER=1` to indicate that we are running the command as `root` https://github.com/composer/composer/blob/666dc93fcc1090efffee82d78e4553caca682087/doc/faqs/how-to-install-untrusted-packages-safely.md?plain=1#L13C19-L13C43

Reply:
Fine by me, i´ve changed it.